# 次のコードの実行結果として正しいものはどれか。

for i in range(-10, -16, -2):
    print(i, end=", ")


# 例外発生

# -12, -14,

# 20, 28

# -10, -12, -14, -16,

# -10, -12, -14,
